packages/dds/sequence/src/test/testFarm.ts

```diff
@@ -13,28 +13,13 @@
 
 import path from "path";
 import { assert, Trace } from "@fluidframework/common-utils";
-import * as MergeTree from "@fluidframework/merge-tree";
-import {
-    TextSegment,
-    createGroupOp,
-    PropertySet,
-    MergeTreeTextHelper,
-    IMergeTreeDeltaOp,
-    ReferenceType,
-} from "@fluidframework/merge-tree";
-import {
-    LocalClientId,
-    NonCollabClient,
-    UnassignedSequenceNumber,
-    UniversalSequenceNumber,
-    // eslint-disable-next-line import/no-internal-modules
-} from "@fluidframework/merge-tree/dist/constants";
 // eslint-disable-next-line import/no-internal-modules
-import { loadTextFromFile, TestClient, TestServer } from "@fluidframework/merge-tree/dist/test/";
+import * as MergeTree from "@fluidframework/merge-tree/dist/test/";
```

Comment:
this is a legacy test that doesn't match the current layering well, just extended the existing submodule import to keep it working. this seems better than exporting everything for everyone